docs/reference/reana-yaml/index.md

```diff
@@ -128,6 +129,11 @@ The **workflow.specification.steps** property describes each individual computat
 | environment | _string_ | _mandatory_ | Specifies runtime environment container image where the given workflow step commands will be run.                                                                                     |
 | commands    | _list_   | _mandatory_ | Lists all commands to be run in the runtime environment container image when the given workflow step is executed. Note that each command is executed as a separate containerised job. |
 
+The **workflow.resources** property is used to describe the resources needed by
+the workflow. For example, see how to set up
```

Comment:
For example, you can use it to set up [CVMFS](/advanced-usage/code-repositories/cvmfs) volume provisioning or
[Kerberos](/advanced-usage/access-control/kerberos/#setting-kerberos-requirement) authentication support for the workflow orchestration.